examples/seismic/tutorials/09_snapshotting.ipynb

```diff
@@ -0,0 +1,806 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snapshotting with Devito using the `ConditionalDimension`\n",
+    "\n",
+    "This notebook intends to introduce new Devito users (especially with a C or FORTRAN background) to the best practice on saving snapshots to disk, as a binary float file. \n",
+    "\n",
+    "We start by presenting a naive approach, and then introduce a more efficient Devito-like method, explaining `ConditionalDimension`.\n",
```

Comment:
... and then introduce a more efficient method, which exploits Devito's `ConditionalDimension`.